libvast/test/system/evaluator.cpp

```diff
@@ -28,13 +30,42 @@ using namespace vast;
 
 namespace {
 
+using counts = std::vector<vast::count>;
+
+template <class F>
+ids select(const counts& xs, vast::count y, F pred) {
+  ids result;
+  for (auto x : xs)
+    result.append_bits(pred(x, y), 1);
```

Comment:
The single-argument function `append_bit` allows you to skip the second argument.